simplivity/resources/omnistack_clusters.py

```diff
@@ -130,9 +130,27 @@ def get_connected_clusters(self):
         connected_clusters = self._client.do_get(method_url).get("omnistack_clusters", [])
 
         clusters = [self._clusters.get_by_id(cluster["id"]) for cluster in connected_clusters]
-
         return clusters
 
+    def get_throughput(self, destination_id=None, time_offset=0, range=43200):
+        """Calculates the throughput between each pair of omnistack_clusters in the federation
+
+Args:
+            destination_id : The unique identifier (UID) of the omnistack_clusters to return
+            time_offset : A time offset in seconds (from now) or a datetime,
+                        expressed in ISO-8601 form, based on Coordinated Universal Time (UTC) Default: 0
+            range : A range in seconds (the duration from the specified point in time) Default: 43200
+
+        Returns:
+            cluster_throughput object.
```

Comment:
Please specify the type of the return object

Reply:
Updated as suggested